srml/support/src/lib.rs

```diff
@@ -1,5 +1,176 @@
+#![recursion_limit = "128"]
 #![cfg_attr(not(feature = "std"), no_std)]
 #![feature(drain_filter)]
 
-pub mod traits;
-pub mod types;
```

Comment:
Recommend to seperate into smaller modules. The support things are unrelated. We provide these to other people to use.

If you want to re-export add a `pub use` or put them in a `prelude` module.